european-journal-of-endocrinology.csl

```diff
@@ -0,0 +1,143 @@
+  <info>
+    <title>European Journal of Endocrinology</title>
+    <title-short>Eur J Endocrinol</title-short>
+    <id>http://www.zotero.org/styles/european-journal-of-endocrinology</id>
+    <link href="http://www.zotero.org/styles/european-journal-of-endocrinology" rel="self"/>
+    <link href="http://www.zotero.org/styles/vancouver" rel="template"/>
+    <link href="http://www.eje-online.org/site/misc/For-Authors.xhtml#refs" rel="documentation"/>
+    <author>
+      <name>Sylvère Störmann</name>
+      <email>sylvere.stoermann@med.uni-muenchen.de</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="medicine"/>
+    <issn>0804-4643</issn>
+    <eissn>1479-683X</eissn>
+    <updated>2016-12-29T23:48:11+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="author">
+    <names variable="author">
+      <name initialize-with="" name-as-sort-order="all" sort-separator=" "/>
+      <substitute>
+        <names variable="editor"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="editor">
+    <group prefix="Eds ">
+      <names variable="editor" suffix=". ">
+        <name and="symbol" initialize-with="" name-as-sort-order="all" sort-separator=" ">
+          <name-part name="given" text-case="uppercase"/>
+        </name>
+      </names>
+    </group>
+  </macro>
+  <macro name="publisher">
+    <group delimiter=": " suffix=",">
+      <text variable="publisher-place"/>
+      <text variable="publisher"/>
+    </group>
+  </macro>
+  <macro name="container-title">
+    <choose>
+      <if type="chapter book" match="any">
+        <group suffix=".">
+          <choose>
+            <if match="any" variable="container-title">
+              <text term="in" text-case="capitalize-first" suffix=" "/>
+              <text variable="container-title" font-style="italic" suffix=","/>
+            </if>
+          </choose>
+          <text macro="edition" prefix=" "/>
+          <choose>
+            <if match="any" variable="chapter-number">
+              <number prefix="ch. " variable="chapter-number" suffix=", "/>
+            </if>
+          </choose>
+          <text macro="pages"/>
+          <text macro="editor"/>
+          <text macro="publisher"/>
+          <text macro="date"/>
+        </group>
+      </if>
+      <else>
+        <group suffix=".">
+          <text variable="container-title" font-style="italic"/>
+          <text macro="date"/>
+          <text macro="journal-location"/>
+          <text macro="pages"/>
+        </group>
+        <text variable="DOI" prefix=" (doi:" suffix=")"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="title">
+    <text variable="title"/>
+  </macro>
+  <macro name="edition">
+    <choose>
+      <if is-numeric="edition">
+        <number prefix="edn " variable="edition" suffix=", "/>
```

Comment:
Sure, no problem!

(just to clarify: were not concerned with this journal changes its publishing language, but with being able to reuse (parts of) this CSL style for publications in other languages, which is easier if the style doesn't use hard-coded English text. But it's not that big of a deal here, so don't worry about it.)